spec/facter/resolvers/augeas_resolver_spec.rb

```diff
@@ -4,9 +4,11 @@
   subject(:augeas) { Facter::Resolvers::Augeas }
 
   let(:log_spy) { instance_spy(Facter::Log) }
+  let(:executable?) { false }
```

Comment:
usually variables do not have `?` in name, that is a convention only for boolean methods

Reply:
changed